README.md

```diff
@@ -86,6 +86,7 @@ COOKIE_SECRET | The shared secret used in validating/calculating the session coo
 INTERNAL_API_URL | Internal API URL | `http://api.url`
 LANDING_PAGE_URL | Register OE landing Page | `/register-an-overseas-entity/sold-land-filter`
 UPDATE_LANDING_PAGE_URL | Update OE landing Page | `/update-an-overseas-entity/overseas-entity-query`
+REMOVE_LANDING_PAGE_URL | Remove OE landing Page | `http://gov-uk-external-remove-guidance-page`
```

Comment:
Do we also need to add an endpoint for remove in the Endpoints section (line 72)?

Reply:
Yes, we should. Will add one and push change to [this](https://github.com/companieshouse/overseas-entities-web/pull/1200) PR.